src/frontend/components/TeacherLiveInfoBar/index.tsx

```diff
@@ -34,7 +34,7 @@ export const TeacherLiveInfoBar = ({
       <Heading
         a11yTitle={liveTitle}
         color="blue-active"
-        level="2"
+        level="5"
         margin={{ bottom: 'small' }}
         size="small"
```

Comment:
```suggestion
        level="1"
        margin={{ bottom: 'small' }}
        size="1.3rem"
```
I think we should use a h1 header, as it is the main title of the view.
Setting the size to 1.3rem allows to get a slightly bigger text than the titles in the blocks below.

before:
![image](https://user-images.githubusercontent.com/720491/165943411-f79dfa60-2fe9-49be-95a0-b75389357a48.png)


after :
![image](https://user-images.githubusercontent.com/720491/165943320-ecefbd48-a1a1-4a11-8eff-7aa1a64eeccb.png)

WDYT?